toolchain/mfc/args.py

```diff
@@ -81,10 +81,11 @@ def add_common_arguments(p, mask = None):
     test.add_argument("-f", "--from",         default=test_cases[0].get_uuid(), type=str, help="First test UUID to run.")
     test.add_argument("-t", "--to",           default=test_cases[-1].get_uuid(), type=str, help="Last test UUID to run.")
     test.add_argument("-o", "--only",         nargs="+", type=str, default=[], metavar="L", help="Only run tests with specified properties.")
-    test.add_argument("-r", "--relentless",   action="store_true", default=False, help="Run all tests, even if multiple fail.")
+    test.add_argument("-r", "--relentless",   action="store_true", default=True, help="Run all tests, even if multiple fail.")
```

Comment:
This is just like removing `--relentless`. That is, there is no way of turning it off through the CLI. We have two options:
- Remove the option entirely.
- Rename it to `--no-relentless` (or some better name) and change action to `store_false` and set default back to `False`.

Reply:
good point, somehow this didn't even cross my mind. but I still agree with you. users can ctrl+c if they see failure and want to quit, as well.